### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by shaojiajun314** [coqui-ai/TTS#4093] [General Q&A]
i deployed tts with scripts
`
python3.10 -m pip install -r requirements.txt
export CUDA_VISIBLE_DEVICES="3"
python3.10 TTS/server/server.py --list_models
python3.10 TTS/server/server.py --model_name tts_models/zh-CN/baker/tacotron2-DDC-GST
`
And then i got very strange sound. Every sound ends with very very long voice (last char), just like 'hello world d d d d d d d d  .......'

**Comment by TheDevloper2023** [comment]
You have to add punctuation to stop the strange sound:
`hello world.`
